insights/debian/control

```diff
@@ -33,3 +33,44 @@ Description: Ubuntu metrics reporting service
  This is designed to be a full replacement for Ubuntu Report.
  .
  This package contains the command line tool and systemd service triggers.
+
+Package: libinsights0
```

Comment:
You are missing a `libinsights0.symbols` which is a requirement for a shared library. You can look at ubuntu-report as an inspiration if needed because the golang generation to have it cross-architecture is not trivial.